--- a/recipes-connectivity/murata-binaries/murata-binaries_1.0.bb	2024-09-13 13:58:55.878632578 +0300
+++ b/recipes-connectivity/murata-binaries/murata-binaries_1.0.bb	2024-09-13 14:43:58.531957646 +0300
@@ -272,7 +272,7 @@
 	  imx7ulpea-ucom)
 		install -m 755 ${S}/switch_module_imx7ulpea-ucom.sh ${D}/usr/sbin/switch_module.sh
 		;;
-	  imx8mmea-ucom)
+	  imx8mm-smart)
 		install -m 755 ${S}/switch_module_imx8mmea-ucom.sh ${D}/usr/sbin/switch_module.sh
 		;;
 	  imx8mnea-ucom)
    
