--- a/meta/recipes-extended/cracklib/cracklib_2.9.7.bb		2024-10-23 06:12:25.566345495 +0000
+++ b/meta/recipes-extended/cracklib/cracklib_2.9.7.bb		2024-10-22 12:36:26.334358578 +0000
@@ -9,11 +9,13 @@
 
 EXTRA_OECONF = "--without-python --libdir=${base_libdir}"
 
-SRC_URI = "git://github.com/cracklib/cracklib;protocol=https;branch=master \
-           file://0001-packlib.c-support-dictionary-byte-order-dependent.patch \
-           file://0002-craklib-fix-testnum-and-teststr-failed.patch"
+SRC_URI = "git://github.com/cracklib/cracklib;protocol=https;branch=main \
+	    file://0001-packlib.patch \
+	    file://0002-teststr.patch \
+	    file://0003-testnum.patch \
+	    "
 
-SRCREV = "f83934cf3cced0c9600c7d81332f4169f122a2cf"
+SRCREV = "81f9ef98ecb90dc295ec2c59d44b2a7ba9335376"
 S = "${WORKDIR}/git/src"
 
 inherit autotools gettext
