--- ./sources/meta-murata-wireless/recipes-connectivity/murata-binaries/murata-binaries_1.0.bb	2023-02-08 13:42:04.272227810 +0000
+++ ./sources/meta-murata-wireless/recipes-connectivity/murata-binaries/murata-binaries_1.2.bb	2023-02-08 13:41:48.868315300 +0000
@@ -218,6 +218,9 @@
 	  imx8mmea-ucom)
 		install -m 755 ${S}/switch_module_imx8mmea-ucom.sh ${D}/usr/sbin/switch_module.sh
 		;;
+	  imx8mm-redge)
+		install -m 755 ${S}/switch_module_imx8mmea-ucom.sh ${D}/usr/sbin/switch_module.sh
+		;;
 	  imx8mnea-ucom)
 		install -m 755 ${S}/switch_module_imx8mnea-ucom.sh ${D}/usr/sbin/switch_module.sh
 		;;
