--- a/poky/meta/recipes-devtools/pseudo/pseudo_git.bb  2026-08-04 15:25:14.055203013 +0300
+++ b/poky/meta/recipes-devtools/pseudo/pseudo_git.bb  2026-05-15 11:15:01.696567203 +0300
@@ -1,6 +1,8 @@
 require pseudo.inc
 
 SRC_URI = "git://git.yoctoproject.org/pseudo;branch=master;protocol=https \
+           file://0001-configure-Prune-PIE-flags.patch \
+           file://glibc238.patch \
            file://fallback-passwd \
            file://fallback-group \
            "
@@ -12,9 +14,9 @@
     file://older-glibc-symbols.patch"
 SRC_URI[prebuilt.sha256sum] = "ed9f456856e9d86359f169f46a70ad7be4190d6040282b84c8d97b99072485aa"
 
-SRCREV = "43cbd8fb4914328094ccdb4bb827d74b1bac2046"
+SRCREV = "0d292df61aeb886ae8ca33d9edc3b6d0ff5c0f0f"
 S = "${WORKDIR}/git"
-PV = "1.9.3+git"
+PV = "1.9.0+git"
 
 # largefile and 64bit time_t support adds these macros via compiler flags globally
 # remove them for pseudo since pseudo intercepts some of the functions which will be
