--- a/common/autoboot.c		2024-08-13 12:46:15.543850618 +0000
+++ b/common/autoboot.c		2024-08-13 13:07:57.279146986 +0000
@@ -368,7 +368,7 @@
 	int abort = 0;
 	unsigned long ts;
 
-	printf("Hit any key to stop autoboot: %2d ", bootdelay);
+//	printf("Hit any key to stop autoboot: %2d ", bootdelay);
 
 	/*
 	 * Check if key already pressed
@@ -376,7 +376,7 @@
 	if (tstc()) {	/* we got a key press	*/
 		getchar();	/* consume input	*/
 		puts("\b\b\b 0");
-		abort = 1;	/* don't auto boot	*/
+		abort = 0;	/* abort key pressing	*/
 	}
 
 	while ((bootdelay > 0) && (!abort)) {
